traffic_portal/app/src/common/modules/table/certExpirations/TableCertExpirationsController.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+var TableCertExpirationsController = function(tableName, certExpirations, dsXmlToIdMap, filter, $document, $scope, $state, $filter, locationUtils) {
+
+	/** All of the expiration fields converted to actual Dates */
+	$scope.certExpirations = certExpirations.map(
+		function(x) {
+			// need to convert this to a date object for ag-grid filter to work properly
+			x.expiration = new Date(x.expiration);
+			return x;
+		});
+
+	$scope.dsXmlToIdMap = dsXmlToIdMap;
+
+	$scope.editCertExpirations = function(dsId) {
+		locationUtils.navigateToPath('/delivery-services/' + dsId + '/ssl-keys');
+	}
+
+	/**
+	 * Formats the contents of a 'federation' column cell as "True" or blank for visibility.
+	 */
+	function federatedCellFormatter(params) {
+		if (!params.value) {
+			return '';
+		} else {
+			return params.value;
+		}
```

Comment:
This will make it always 'true' or blank - shouldn't it be 'true' or 'false'?

Reply:
yea i put it as 'true' or blank because when it was 'true' or 'false' it was not super clear at just a glance. so this way its easy to see quickly

Reply:
Okay, that's fine with me. You could also change it to a checkmark or something if you like that better - I think the cache checks page has an example of that - but as far as I'm concerned this is fine.